lib/acts_as_paranoid/validations.rb

```diff
@@ -38,8 +37,11 @@ def validate_each(record, attribute, value)
         protected
 
         def build_relation(klass, attribute, value)
-          return super(klass, klass.arel_table, attribute, value) if ActiveRecord::VERSION::MINOR == 0
-          super
+          if ActiveRecord::VERSION::MINOR == 0 && ActiveRecord::VERSION::MAJOR == 5
```

Comment:
Agreed.  I wasn't sure what the best way to code this would be since most options seemed to involve just copying and pasting, so I opted for the fewest lines of code: simply making V6 an empty subclass of V5.